google/resource_dataproc_cluster.go

```diff
@@ -310,6 +310,21 @@ func resourceDataprocCluster() *schema.Resource {
 								},
 							},
 						},
+						"encryption_config": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
```

Comment:
I'm not sure this should be `Computed`. We normally only tag a field as `Computed` if it's going to have something coming back from the api when it hasn't originally been passed in the original request or if it's a completely calculated field in the upstream api.

When I tested the GET api on a simple cluster it didn't show anything coming back for `encryption_config`. Since it doesn't appear to be a calculated or output style field I think somebody should explicitly have this in their config if the cluster has encryption on it. I think that `Optional` but not `Computed` is the right choice here.